README.md

```diff
@@ -0,0 +1,53 @@
+#DateBuilder#
```

Comment:
Incorrect markdown syntax: `#DateBuilder#` -> `# DateBuilder`

Reply:
ok fixed in [commit](https://github.com/LaRoueVerte/datebuilder/pull/1/commits/5c0622b4ebb5b9a952b21e0002ee024a104be21c)